src/main/java/io/blamer/bot/conversation/routes/ChatId.java

```diff
@@ -0,0 +1,67 @@
+package io.blamer.bot.conversation.routes;
+
+import io.blamer.bot.conversation.Conversation;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.commands.BotCommand;
+import reactor.core.publisher.Mono;
+
+import javax.naming.OperationNotSupportedException;
+
+/*
+ * @todo #73:45min\DEV Create tests for ChatId
+ */
+/**
+ * Chat id command.
+ */
+@Component("/chatid")
+public class ChatId implements Conversation {
```

Comment:
@l3r8yJ maybe a better name for this class will be RegisterChat?

Reply:
@h1alexbel Honestly, I'd rather make this command like `/unsubscribe` because we already have `/register`

Reply:
@l3r8yJ we don't have register chat id command.
We have registry, but it's a different flow

Reply:
@h1alexbel I think you can get by with a registry, storing the `chat id` + `token` at the same time

Reply:
@l3r8yJ you can, but still now we have a separate flow for it, simply because register chat is like user registration in blamer, while register token is actually different flow

What do you think?

Reply:
@h1alexbel I think that if the token is successfully authorized we can also save the chat id, wdyt?

Reply:
@l3r8yJ you mean combining two flows together?
If so, sounds good, here are my suggestions:
1. create a /token command where we can pass token or token + chat id
2. create a /chatid command where we can just persist chat id
3. and having /unsubscribe to remove chat id from hub database

2nd flow will be useful for users who just want to wire chat without adding token right now

WDYT?

Reply:
@l3r8yJ thus, we will have the following commands on bot:
* /chatid
* /token
* /unsubscribe

Or I miss something?

Reply:
@h1alexbel рonestly, I don't see any cases where a user would want to save a chat without receiving notifications

what's the motivation behind it?

Reply:
@l3r8yJ actually yes, agree.
So we will reduce amount of commands simply to: /unsub and /token. On token we will validate if the chat already exists in db

Reply:
@l3r8yJ can you apply these changes to diagrams in order to sync up

Reply:
@h1alexbel i'll create a puzzle

Reply:
Ok